plugins/volumeFilter_test.go

```diff
@@ -145,3 +146,229 @@ func TestMakeFilterVolume(t *testing.T) {
+			mode:               volumeFilterModeIgnore,
+			sellBaseCapInBase:  pointy.Float64(1.0),
+			sellBaseCapInQuote: nil,
+			otbBase:            pointy.Float64(0.0),
+			otbQuote:           pointy.Float64(0.0),
+			tbbBase:            pointy.Float64(0.0),
+			tbbQuote:           pointy.Float64(0.0),
+			inputOp:            makeManageSellOffer("2.0", "100.0"),
+			wantOp:             nil,
+			wantTbbBase:        pointy.Float64(0.0),
+			wantTbbQuote:       pointy.Float64(0.0),
+		},
+		{
+			name:               "5. selling, base units sell cap, keep selling base, exact mode",
+			mode:               volumeFilterModeExact,
+			sellBaseCapInBase:  pointy.Float64(1.0),
+			sellBaseCapInQuote: nil,
+			otbBase:            pointy.Float64(0.0),
+			otbQuote:           pointy.Float64(0.0),
+			tbbBase:            pointy.Float64(0.0),
+			tbbQuote:           pointy.Float64(0.0),
+			inputOp:            makeManageSellOffer("2.0", "100.0"),
+			wantOp:             makeManageSellOffer("2.0", "1.0000000"),
+			wantTbbBase:        pointy.Float64(1.0),
+			wantTbbQuote:       pointy.Float64(2.0),
+		},
+		{
+			name:               "6. selling, base units sell cap, keep selling base, ignore mode",
+			mode:               volumeFilterModeIgnore,
+			sellBaseCapInBase:  pointy.Float64(1.0),
+			sellBaseCapInQuote: nil,
+			otbBase:            pointy.Float64(0.0),
+			otbQuote:           pointy.Float64(0.0),
+			tbbBase:            pointy.Float64(0.0),
+			tbbQuote:           pointy.Float64(0.0),
+			inputOp:            makeManageSellOffer("2.0", "100.0"),
+			wantOp:             nil,
+			wantTbbBase:        pointy.Float64(0.0),
+			wantTbbQuote:       pointy.Float64(0.0),
+		},
+		{
+			name:               "7. selling, quote units sell cap, don't keep selling quote, exact mode",
+			mode:               volumeFilterModeExact,
+			sellBaseCapInBase:  nil,
+			sellBaseCapInQuote: pointy.Float64(0),
+			otbBase:            pointy.Float64(0.0),
+			otbQuote:           pointy.Float64(0.0),
+			tbbBase:            pointy.Float64(0.0),
+			tbbQuote:           pointy.Float64(0.0),
+			inputOp:            makeManageSellOffer("2.0", "100.0"),
+			wantOp:             nil,
+			wantTbbBase:        pointy.Float64(0.0),
+			wantTbbQuote:       pointy.Float64(0.0),
+		},
+		{
+			name:               "8. selling, quote units sell cap, don't keep selling quote, ignore mode",
+			mode:               volumeFilterModeIgnore,
+			sellBaseCapInBase:  nil,
+			sellBaseCapInQuote: pointy.Float64(0),
+			otbBase:            pointy.Float64(0.0),
+			otbQuote:           pointy.Float64(0.0),
+			tbbBase:            pointy.Float64(0.0),
+			tbbQuote:           pointy.Float64(0.0),
+			inputOp:            makeManageSellOffer("2.0", "100.0"),
+			wantOp:             nil,
+			wantTbbBase:        pointy.Float64(0.0),
+			wantTbbQuote:       pointy.Float64(0.0),
+		},
+		{
+			name:               "9. selling, quote units sell cap, keep selling quote, new amount, exact mode",
```

Comment:
seems same as # 11 below?
what do you mean by "new amount" in the name? -- is that a remnant of the old set of test cases?